Dockerfiles/manifests/rbac/clusterrole.yaml

```diff
@@ -1,55 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
please dont delete the `Dockerfiles/manifests/rbac` files since they are used by the documentation: https://docs.datadoghq.com/agent/kubernetes/?tab=daemonset